jetbrains-core/src/software/aws/toolkits/jetbrains/settings/AwsSettingsConfigurable.form

```diff
@@ -8,7 +8,7 @@
     <properties/>
     <border type="none"/>
     <children>
-      <grid id="cb6a8" binding="serverlessSettings" layout-manager="GridLayoutManager" row-count="2" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+      <grid id="cb6a8" binding="serverlessSettings" layout-manager="GridLayoutManager" row-count="1" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```

Comment:
What change is this representing?

Reply:
This is the removal of the Lambda gutter-icon preference which is the only one stored at the project level.